<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEDBrowserUI</class>
 <widget class="QMainWindow" name="ZEDBrowserUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="Layout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="LeftLayout"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="RightLayout">
        <item>
         <widget class="QScrollArea" name="BrowserScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>694</width>
             <height>618</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="SmallAction">
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="MediumAction">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="LargeAction">
   <property name="text">
    <string>Large</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="CopyAction">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="PasteAction">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="CutAction">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="DeleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="EditAction">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="NewResourceActon">
   <property name="text">
    <string>New Resource</string>
   </property>
  </action>
  <action name="SelectAllAciton">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="DesellectAction">
   <property name="text">
    <string>Desellect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
